static/linker/demo-page.html

```diff
@@ -0,0 +1,87 @@
+<html ⚡️>
+<head>
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script>
```

Comment:
This confuses me. Can you explain why it's needed?

Reply:
It's not. Good catch. Deleted.